package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stytch",
-  "version": "3.3.0",
+  "version": "3.4.0",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
```

Comment:
CI is complaining about line 9 in this file. I can't comment directly because it's unchanged in this PR, which is ironic, because that's exactly the bug 😛 

This should get automatically updated when you run `npm install` (to update line 3). So I think this might be a difference in Node or NPM versions. Here are mine that seem to match CI when it checks:
```
$ node --version
v16.9.0
$ npm --version
7.21.1
```

Reply:
Updated the version and that seems to fix the issue